business/definitionService.js

```diff
@@ -115,7 +116,10 @@ class DefinitionService {
     // If no tools participated in the creation of the definition then don't bother storing.
     // Note that curation is a tool so no tools really means there the definition is effectively empty.
     const tools = get(definition, 'described.tools')
-    if (!tools || tools.length === 0) return definition
+    if (!tools || tools.length === 0) {
+      await this.harvestService.harvest({ tool: 'package', coordinates })
```

Comment:
In GHCrawler if the same thing was queued multiple times the subsequent queue requests were dropped, is it the same in ClearlyDefined?

Reply:
I'm guessing that is the case, but I'm not sure. It's using a branch of GHCrawler
https://github.com/Microsoft/ghcrawler/tree/jm/apicrawler/providers/queuing

Do you know where that handling was done?